lte/gateway/deploy/iperfserver.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Why do we need an additional file when there is already `magma_trfserver.yml`?

Reply:
The ```magma_trfserver.yml``` uses the host ```trfserver``` with the username ```vagrant```, while the iperfserver.yml is using the iperfserver host with the username ```ubuntu```.

Reply:
We can keep the magma_trfserver.yml and create a vagrant username on AWS instance? but does it make sense?

Reply:
@frmess we can keep only one file in the repository and make the changes at runtime in `hosts` file, also the ip-address in the `hosts` file depends on the AWS instances which is created.